package.json

```diff
@@ -7,19 +7,18 @@
   ],
   "packageManager": "yarn@1.22.19",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.60.1",
-    "@typescript-eslint/parser": "^5.60.1",
-    "eslint": "^8.43.0",
+    "@eslint/js": "^9.2.0",
+    "@typescript-eslint/eslint-plugin": "^7.8.0",
+    "@typescript-eslint/parser": "^7.8.0",
```

Comment:
I don't think we need these anymore, right? Just `typescript-eslint`?

Reply:
Oh yes, I forgot to delete them, I also removed `eslint-config-prettier` . 
I'll check the default rules that are used but I believe that stylistic rules that are handled by prettier are removed from recommended `eslint` and `ts-eslint` rules.